iphone/Classes/AppModule.m

```diff
@@ -314,11 +314,12 @@ -(void)keyboardFrameChanged:(NSNotification*) notification
     }
     
     NSDictionary *event = [NSDictionary dictionaryWithObjectsAndKeys:
-                                [TiUtils rectToDictionary:keyboardEndFrame], @"keyboardFrame",
-                                nil];
+                           [TiUtils rectToDictionary:keyboardEndFrame], @"keyboardFrame",
+                           duration, @"animationDuration",
+                           nil];
+    
     
-    [self fireEvent:@"keyboardFrameChanged" withObject:event]; 
-    [self fireEvent:@"keyboardframechanged" withObject:event];     
+    [self fireEvent:@"keyboardframechanged" withObject:event];
```

Comment:
The event `keyboardframechanged` is documented as `keyboardFrameChanged` (camel cased)